docs/mllib-linear-methods.md

```diff
@@ -185,10 +185,10 @@ algorithm for 200 iterations.
 import org.apache.spark.mllib.optimization.L1Updater
 
 val svmAlg = new SVMWithSGD()
-svmAlg.optimizer.
-  setNumIterations(200).
-  setRegParam(0.1).
-  setUpdater(new L1Updater)
+svmAlg.optimizer
```

Comment:
Yeah I see the change, just wondering about why. OK it's just for consistency? I searched and it does seem more frequently written with the leading dot. OK.